src/components/SignInOrJoinFree.js

```diff
@@ -36,7 +36,7 @@ class SignInOrJoinFree extends React.Component {
   };
 
   state = {
-    form: 'signin',
+    form: this.props.form || 'signin',
```

Comment:
Can we rename that `initialForm`? The current docs is confusing because this changes makes the prop un-controlled (updating the prop won't update the component).

```
Use this prop to use this as a controlled component
```

Reply:
sure thing